docs/manage-default-themes-and-styles.md

```diff
@@ -0,0 +1,157 @@
+# Themes & Styles
+
+This help topic describes how to add SurveyJS themes to your Angular, Vue, React, Knockout, or jQuery application and switch between them if you want to apply multiple themes.
+
+## Add SurveyJS Themes to Your Application
+
+SurveyJS ships with the Modern and Default V2 UI themes illustrated below.
```

Comment:
"V2" was added to distinguish the new version of the Default theme from the old one. The old one is now obsolete, but the name stuck. As a rule, we don't enclose names in quotes because in most cases capitalization is enough.

Reply:
OK, I see your point. Thank you for the clarification.